src/nupic/proto/SpatialPoolerProto.capnp

```diff
@@ -26,7 +26,7 @@ struct SpatialPoolerProto {
   minPctOverlapDutyCycles @15 :Float32;
   minPctActiveDutyCycles @16 :Float32;
   dutyCyclePeriod @17 :UInt16;
-  maxBoost @18 :Float32;
+  boostStrength @18 :Float32;
```

Comment:
If you don't care about backwards compatibility then you can do whatever you want as long as what left uses incrementing, sequential numbers (don't have to be in order in the file though).

It doesn't really cost anything to keep backwards compatibility so if you want to get in that mindset, here are the rules:

- Never remove a field, just add a comment saying that it is no longer used
- Don't change field types, mark the current field as obsolete and create a new one with the new field type

But we've stated that we aren't keeping backward compatibility so go ahead and just redo the schema as you see fit. Then we need to announce that with the release as @rhyolight outlined here: https://discourse.numenta.org/t/introducing-breaking-changes/1647